StudyPathTraker/StudyPathTraker/Modules/CheckList/New CheckList/SPNewCheckListPresenter.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SPNewCheckListPresenter.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 6/19/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+protocol SPNewCheckListPresenterProtocol: SPBasePresenterProtocol {
+    var presenter: SPNewCheckListPresenter { get }
+}
+
+class SPNewCheckListPresenter {
+
+    weak var delegate: SPNewCheckListPresenterProtocol?
+
+    func addMilestone(_ milestone: Milestone, item: Item) {
+        let result = PersistenceManager.updateItem {
```

Comment:
why you are doing this? just save the object. Realm is smart enough for update when the object has an existing primary key.